.github/workflows/manual-perf.yml

```
@@ -93,5 +93,6 @@ jobs:
       specific_benchmarks: ${{ github.event.inputs.specific_benchmarks }}
       vector_type: ${{ github.event.inputs.vector_type }}
       ddr_config: ${{ github.event.inputs.ddr_config }}
+      extra_args: ${{ github.event.inputs.extra_args }}
```

actionlint (1.7.12)

[error] property "extra_args" is not defined in object type {benchmark_type: string; branch: string; configuration: string; ddr_config: string; note: string; specific_benchmarks: string; vector_type: string}

(expression)